account_analytic_distribution/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
With Python3 the coding utf-8 lines can be removed.

Reply:
@danger89 Thank you, i made the changes, could you check please.